phoenix-pherf/src/it/java/org/apache/phoenix/pherf/PherfMainIT.java

```diff
@@ -23,6 +23,7 @@
 import org.apache.phoenix.pherf.result.ResultValue;
 import org.apache.phoenix.pherf.result.file.ResultFileDetails;
 import org.apache.phoenix.pherf.result.impl.CSVFileResultHandler;
+import org.junit.Ignore;
```

Comment:
nit: can you remove this unused import from the next PR. Don't need to fix it now